CMakeLists.txt

```diff
@@ -110,10 +110,11 @@ if(NOT UMF_LINK_HWLOC_STATICALLY)
             "${DLL_PATH_LIST};PATH=path_list_append:${LIBHWLOC_LIBRARY_DIRS}/../bin"
         )
     endif()
-else()
-    if(NOT WINDOWS)
-        message(FATAL_ERROR "hwloc can be statically linked only on Windows")
-    endif()
+    # add PATH to DLL on Windows
+    set(DLL_PATH_LIST
+        "${DLL_PATH_LIST};PATH=path_list_append:${LIBHWLOC_LIBRARY_DIRS}/../bin"
+    )
```

Comment:
if this flag works on non-Windows OSes, please update the `UMF_LINK_HWLOC_STATICALLY` description in CMake and in README

Reply:
Done.